src/components/macro/macro.component.ts

```diff
@@ -1,12 +1,132 @@
-import { Component } from '@angular/core';
+import { Macro } from '../../config-serializer/config-items/Macro';
```

Comment:
Move angular imports to the first block, rxjs import to the second, dragula the third, rest is the fourth.